index.html

```diff
@@ -27,9 +27,8 @@
 <body>
 	<header>
 		<h1>Learn Teach Code</h1>
-		<h5><a href="https://learnteachcode.org/slack" target="_blank" rel="noopener">Join our Slack!</a> | <p>
-			<strong><a href="https://discord.gg/zSEyPCsFWD" target="_blank" rel="noopener">Join our Discord!</a></strong>
-		</p> | <a href="https://learnteachcode.org/code-coffee-compendium" target="_blank" rel="noopener">Resources</a> | <a href="./code-of-conduct.html">Code of Conduct</a></h5>
+		<h5><a href="https://learnteachcode.org/slack" target="_blank" rel="noopener">Join our Slack!</a> |
```

Comment:
I can suggest a solution with javascript, as the template is same and just the links change, we can just make an object which contains the href links along with the title to display and just render it through.

Reply:
Thanks for the suggestion @anarchymonkey ! We're due for a total site overhaul anyhow so I'm going to leave it as is for now and later if we pick a blogging template (maybe Jekyll since it works well with GitHub Pages already, or something newer/trendier perhaps) it'll have support for templating via the static site generator :) and that way we won't have to run any client-side code to do templating which would be lovely